scripts/protocgen.sh

```diff
@@ -17,7 +17,7 @@ protoc_gen_gocosmos() {
 
 protoc_gen_gocosmos
 
-proto_dirs=$(find ./proto -path -prune -o -name '*.proto' -print0 | xargs -0 -n1 dirname | sort | uniq)
+proto_dirs=$(find ./proto -path -prune -o -name '*.proto' -not -path '*/v2alpha2/*' -print0 | xargs -0 -n1 dirname | sort | uniq)
```

Comment:
Thanks for making this script! For now, I'll apply this script in the branch #360 as well.

Reply:
I think you should change it below. :)
```suggestion
proto_dirs=$(find ./proto -path -prune -o -name '*.proto' -not -path '*/v2alpha1/*' -print0 | xargs -0 -n1 dirname | sort | uniq)
```

Reply:
I think you should change it below :)
```suggestion
proto_dirs=$(find ./proto -path -prune -o -name '*.proto' -not -path '*/v2alpha1/*' -print0 | xargs -0 -n1 dirname | sort | uniq)
```

We need to */alpha2/* packages.

Reply:
And if you run proto-gen after making this change, there are a few things that need to be corrected.

Reply:
You're right.
As I mentioned in description, `proto-gen` also generates pb.go files under the same package.
So this change was for success of `build-all` and `test` in github actions.

When we develop doracle, version of `v2alpha2`, we should change the `protogen.sh` as you suggested.

Reply:
Oh man, I understood that I was ignoring `v2alpha1`.
I'm sorry :)

Reply:
No problem :)
Let's ignore `v2alpha1` in the next PR with new ver(`v2alpha2`) of doracle!